src/option.cpp

```diff
@@ -898,9 +940,29 @@ void process_option_line(const std::string &config_line, const char *filename)
          }
       }
    }
+   else if (cmd == "using")
```

Comment:
I am not entirely sure about the `using` keyword here. I see a small possibility for confusion as there are already concepts bound to that keyword in languages that we are formatting.